src/containers/shared/components/Transaction/NFTokenCancelOffer/Simple.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+
+import { useTranslation } from 'react-i18next'
+import Account from '../../Account'
+import { SimpleRow } from '../SimpleRow'
+import { Amount } from '../../Amount'
+import { NFTokenCancelOfferInstructions } from './types'
+import { TransactionSimpleComponent, TransactionSimpleProps } from '../types'
+
+export const Simple: TransactionSimpleComponent = ({
+  data,
+}: TransactionSimpleProps<NFTokenCancelOfferInstructions>) => {
+  const { cancelledOffers } = data?.instructions
+  const { t } = useTranslation()
+
+  return (
+    <>
+      {cancelledOffers.map(({ amount, offerID, tokenID, offerer }) => (
```

Comment:
This might be a good place to reuse the attestation styles @mvadari added.

I wonder if this looks confusing with multiple offers cancelled. @bugsbunnies

Reply:
What does it look like in the UI? @ckniffen

Reply:
I will try to craft an example

Reply:
@bugsbunnies https://nft-devnet.xrpl.org/transactions/3A0452F7893ADBE2D11492BA6960DE0A3C8063FB2D900C62FF885E37B404C291

Reply:
Created #374

Reply:
Thanks @ckniffen